sdk/search/azure-search-documents/tests/test_service_live.py

```diff
@@ -209,6 +295,26 @@ def test_delete_synonym_map(self, api_key, endpoint, index_name, **kwargs):
         client.delete_synonym_map("test-syn-map")
         assert len(client.get_synonym_maps()) == 0
 
+    @SearchResourceGroupPreparer(random_name_enabled=True)
+    @SearchServicePreparer(schema=SCHEMA, index_batch=BATCH)
+    def test_delete_synonym_map_if_unchanged(self, api_key, endpoint, index_name, **kwargs):
+        client = SearchServiceClient(endpoint, AzureKeyCredential(api_key)).get_synonym_maps_client()
+        result = client.create_synonym_map("test-syn-map", [
+            "USA, United States, United States of America",
+            "Washington, Wash. => WA",
+        ])
+        sm_result = self._build_synonym_map_from_dict(result)
+        etag = sm_result.e_tag
+
+        client.create_or_update_synonym_map("test-syn-map", [
+                    "Washington, Wash. => WA",
+                ])
```

Comment:
funky indentation :)

Reply:
cons of copy pasting from previous test :)